src/components/SimpleChart.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  ColorType,
+  createChart,
+  type ISeriesApi,
+  type SingleValueData,
+  type Time,
+} from 'lightweight-charts'
+import { useEffect, useRef } from 'react'
+import { selectSelectedCurrency } from 'state/slices/selectors'
+import { store } from 'state/store'
+
+type SimpleChartProps<T extends number | Time> = {
+  data: SingleValueData<T>[]
+  seriesType: 'histogram' | 'line'
+}
+
+const backgroundColor = 'rgba(188, 214, 240, 0.04)'
+const lineColor = '#2962FF'
+const textColor = 'white'
+const areaTopColor = 'rgba(41, 98, 255, 0.5)'
+const areaBottomColor = 'rgba(41, 98, 255, 0.28)'
+
+const currentLocale = window.navigator.languages[0]
+const selectedCurrency = selectSelectedCurrency(store.getState())
+const priceFormatter = Intl.NumberFormat(currentLocale, {
+  style: 'currency',
+  currency: selectedCurrency,
+  notation: 'compact',
+  compactDisplay: 'short',
+}).format
+
+const chartContainerStyles = { width: '100%', height: '500px' }
+
+export const SimpleChart = <T extends number | Time>({ data, seriesType }: SimpleChartProps<T>) => {
```

Comment:
can just extend `Time` now that `UTCTimestamp` is being used to keep it in alignment with the lib api
```suggestion
export const SimpleChart = <T extends Time>({ data, seriesType }: SimpleChartProps<T>) => {
```

Reply:
4437dec80e5